python/packages/autogen-ext/src/autogen_ext/models/_ollama/_ollama_client.py

```diff
@@ -0,0 +1,144 @@
+import json
+import os
+import base64
+from dataclasses import dataclass
+from typing import Any, Dict, List, Literal, Optional, Union
+
+import aiohttp
+from autogen_core.components import Image
+from autogen_core.components.models import (
+    ChatCompletionClient,
+    ModelCapabilities,
+    AssistantMessage,
+    SystemMessage,
+    UserMessage,
+)
+
+@dataclass
+class OllamaConfig:
+    base_url: str = "http://localhost:11434"
+    model: str = "llama3.2-vision"
+    temperature: float = 0.7
+    top_p: float = 0.95
+
+class OllamaChatCompletionClient(ChatCompletionClient):
+    def __init__(
```

Comment:
Doc string like other clients

Reply:
Also, instead of using OllamaConfig, can we use keyword arguments like other clients?